helm-chart/renku-notebooks/templates/ingress.yaml

```diff
@@ -35,5 +35,9 @@ spec:
             backend:
               serviceName: {{ $fullName }}
               servicePort: http
+          # - path: /jupyterhub
+          #   backend:
+          #     serviceName: proxy-public
+          #     servicePort: 8000
```

Comment:
You should probably uncomment this

Reply:
well, the ingress is handled by renku. And when you run from minikube, it doesn't use the ingress, just a `NodePort`.